packages/math/unicode-symbols.lua

```diff
@@ -2649,6 +2649,9 @@ symbolDefaults["]"] = { atom = atomType.closeSymbol, stretchy = true }
 symbolDefaults["{"] = { atom = atomType.openingSymbol, stretchy = true }
 symbolDefaults["}"] = { atom = atomType.closeSymbol, stretchy = true }
 symbolDefaults["mod"] = { atom = atomType.binaryOperator }
+symbolDefaults["lim"] = { atom = atomType.bigOperator }
```

Comment:
I would have preferred not doing this... But above, replacing `\def{lim}{\mo{lim}}` by `\def{lim}{\mo[atom=big]{lim}} crashes.
... At the same sup/sub stuff as https://github.com/sile-typesetter/sile/issues/2130#issuecomment-2419679652

Sounds there's something deeper to it...

Reply:
And suddenly: Closes #2130 

The many Gods of Typography are with me tonight?

Reply:
![image](https://github.com/user-attachments/assets/fd3eddef-8e9d-4f15-ba89-4babe9ece40c)